quill-cassandra-ce/src/main/scala-2.12/io/getquill/util/GuavaCeUtils.scala

```diff
@@ -0,0 +1,28 @@
+package io.getquill.util
+
+import com.google.common.util.concurrent.{ FutureCallback, Futures, ListenableFuture }
+
+import cats.effect.Async
+import scala.concurrent.ExecutionContext.global
+
+import java.util.concurrent.Executor
+import scala.language.higherKinds
+
+object GuavaCeUtils {
```

Comment:
Why is this called `GuavaCeUtils`, specifically why is the `Guave` here?

Reply:
That object hold  and implicit class to convert guava `ListenableFuture[T]` to the `F[T]`. What do you think will be an approtiate name for it?

Reply:
I think this is reasonable. The Cassandra session code uses Guava's Future to do Async so we have to deal with Guava in any case.

Reply:
This is fine. The Cassandra session directly uses Guava Futures so you have no choice but to wrap that stuff into `F[_]` and this is a reasonable implementation of that.

Reply:
Ah I didn't realize that cassandra uses Guave futures, in that case its okay. I still personally think its better design to put this inside of `CassandraCeContext` or `CeContext` rather than making a new top level object since this is consistent with how the rest of quill works.